configs/triton_services/aws.yaml

```diff
@@ -0,0 +1,7 @@
+head_node_type:
```

Comment:
is there a reason these are using the old CLI instead of the new one? https://docs.anyscale.com/reference/compute-config-api

Reply:
Mostly just me don't know that changed 😅 Will update these